src/constants/colors.ts

```diff
@@ -1,8 +1,9 @@
 export const blue: string = '#288CCB';
-export const green: string = '#53E69D';
+export const green: string = '#5CB85C';
```

Comment:
add a TODO, andy's PR already added this color

Reply:
added TODO with green color idk what its correct